src/repl/QuackRepl.php

```diff
@@ -70,6 +70,14 @@ function start_repl()
     repl();
 }
 
+function quack()
+{
+    if (isPOSIX()) {
```

Comment:
only for the REPL, though, right? Could we maybe pass a parameter to shut it down?